src/ckeditor.jsx

```diff
@@ -145,6 +145,7 @@ export default class CKEditor extends React.Component {
 	async _initializeEditor() {
 		await this.editorDestructionInProgress;
 
+		/* istanbul ignore next */
 		if ( this.watchdog ) {
 			return;
 		}
```

Comment:
It seems that this condition is no longer necessary. It was added while we were working on double rendering issue. But I leave it just in case.